### PR TITLE
Solving #58

### PR DESCRIPTION
**Pull request by iblancasa**
Now, you can run test with "gulp test"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +80,7 @@
     "router"
   ],
   "scripts": {
-    "test": "./test/test.sh",
+    "test": "./node_modules/.bin/gulp test",
```
Comment (jsdario) [inline package.json]: This is a nice change! Thanks. Do you mind if I commit tomorrow further? I started thinking in changing this line to:
`"test": "export NODE_PATH=.:$NODE_PATH; mocha --bail --recursive"`

This way we can run `npm test`, we can get rid of `gulp-mocha` and is not necessary relative paths down there!

File: test/api/resources.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ require('dotenv').load()
 var should = require('chai').should()
 var expect = require('chai').expect
 
-var Resource = require('src/models/resource')
-var helper = require('src/helpers/resource')
+var Resource = require('../../src/models/resource')
+var helper = require('../../src/helpers/resource')
```
Comment (jsdario) [inline test/api/resources.js]: Here!  With `"test": "export NODE_PATH=.:$NODE_PATH; mocha --bail --recursive"`
We can keep absolute paths thanks to `NODE_PATH=.:$NODE_PATH`, though it is a trivial change.

**Comment by jsdario** [comment]
Thanks @iblancasa for your PR I will be merging tomorrow :+1:

**Comment by iblancasa** [comment]
You are welcome.
How about add modify the environment var in the gulp code?

**Comment by jsdario** [comment]
That is possible too, I'll make a branch and comment out a reference to you so you decide. Thanks again :smile:

**Pull request closed without merging**